Utilities/StrFmt.cpp

```diff
@@ -374,6 +374,13 @@ std::string fmt::replace_all(const std::string& src, const std::string& from, co
 	return target;
 }
 
+std::string fmt::replace_all_fix_path(const std::string& src, const std::string& from, const std::string& to)
```

Comment:
Since the caller isn't specifying any replacement, I think the name can be shortened to just fmt::fix_path. The replace_all part is implied.